README.md

```diff
@@ -1,8 +1,274 @@
-# RFC
+# Amethyst RFCs
 
-This is the repository where we work and keep track of the all RFC documents related to Amethyst.
-Having this repository allows people to participate in decision making by opening a pull request with a new document.
+[Amethyst RFCs]:
 
-## Creating a new RFC
+Note that this will be actual links once the bot is set up
```

Comment:
Should we even mention this before the bot is done?

Reply:
Yes, or I'll forget

Reply:
Hmm, at some point I commented that we should have three links here. I'm not sure where that comment is now, but... 😅 I think we can just add them now, independently ot the bot.